src/controllers/physicalCustomer.controller.js

```diff
@@ -0,0 +1,76 @@
+import {
+  createPhysicalCustomerRepository,
+  deletePhysicalCustomerRepository,
+  getAllPhysicalCustomers,
+  getPhysicalCustomer,
+  updatePhysicalCustomerRepository
+} from '../repositories/physicalCustomer.repository.js'
+
+export const getPhysicalCustomers = async (req, res) => {
```

Comment:
req, res ? Use better name description. (request, response) sound better

Reply:
Those names are convention of the library of express, please read the documentation https://expressjs.com/es/guide/routing.html

> req, res ? Use better name description. (request, response) sound better